src/bridge/cockpitpeer.c

```diff
@@ -478,10 +480,19 @@ on_transport_control (CockpitTransport *transport,
           forward = handled = g_hash_table_remove (self->authorizes, cookie);
         }
     }
+  else if (g_str_equal (command, "logout"))
+    {
+      forward = TRUE;
+      /* If this is a privileged bridge, let's not even try to start
+       * it after a "logout" message.  We know that it will likely
+       * fail, but at least for sudo it will fail very slowly.
+       */
+      if (cockpit_json_get_bool (self->config, "privileged", FALSE, &privileged) && privileged)
+        self->poisoned = TRUE;
```

Comment:
Would be nice to `g_warning()` here, as a failing `cockpit_json_get_bool` means that we botched a manifest and the value is not a bool.

Reply:
What about adding the warning to `cockpit_json_get_bool` (etc) itself?

Reply:
Because those functions are used in scenarios where they do not know the actual type, and don't expect a warning. Not a bad idea ... but if you do this, I imagine it would be a follow up with significant line changes.

Reply:
> Would be nice to g_warning() here

We are not doing that for other pieces of `self->config`, so...

Reply:
OK. Not important, but I thought it would be a bit more helpful than just silently ignoring a setting. I'm fine with ignoring this for now.